mycli/main.py

```diff
@@ -740,11 +750,13 @@ def get_prompt(self, string):
               help='Enable/disable LOAD DATA LOCAL INFILE.')
 @click.option('--login-path', type=str,
               help='Read this path from the login file.')
+@click.option('--execute', type=str, help='Execute query to the database.')
```

Comment:
I realize I've already merged this, but this should also include a short option flag for `-e`. Would you mind adding that?